sites/ademe/src/backend/ademe/settings.py

```diff
@@ -0,0 +1,755 @@
+        "djangocms_picture",
+        "djangocms_text_ckeditor",
+        "djangocms_video",
+        "cms",
+        "menus",
+        "sekizai",
+        "treebeard",
+        "filer",
+        "easy_thumbnails",
+        # Django
+        "django.contrib.auth",
+        "django.contrib.contenttypes",
+        "django.contrib.sessions",
+        "django.contrib.admin",
+        "django.contrib.sites",
+        "django.contrib.sitemaps",
+        "django.contrib.staticfiles",
+        "django.contrib.messages",
+    )
+
+    # Languages
+    # - Django
+    LANGUAGE_CODE = "en"
+
+    # Careful! Languages should be ordered by priority, as this tuple is used to get
+    # fallback/default languages throughout the app.
+    # Use "en" as default as it is the language that is most likely to be spoken by any visitor
+    # when their preferred language, whatever it is, is unavailable
+    LANGUAGES = (("en", _("English")), ("fr", _("French")))
+
+    # - Django CMS
+    CMS_LANGUAGES = {
+        "default": {
+            "public": True,
+            "hide_untranslated": False,
+            "redirect_on_fallback": True,
+            "fallbacks": ["en", "fr"],
+        },
+        1: [
+            {
+                "public": True,
+                "code": "en",
+                "hide_untranslated": False,
+                "name": _("English"),
+                "fallbacks": ["fr"],
+                "redirect_on_fallback": False,
+            },
+            {
+                "public": True,
+                "code": "fr",
+                "hide_untranslated": False,
+                "name": _("French"),
+                "fallbacks": ["en"],
+                "redirect_on_fallback": False,
+            },
+        ],
+    }
+
+    # - Django Parler
+    PARLER_LANGUAGES = CMS_LANGUAGES
+
+    # Permisions
+    # - Django CMS
+    CMS_PERMISSION = True
+
+    # - Django Filer
+    FILER_ENABLE_PERMISSIONS = True
+    FILER_IS_PUBLIC_DEFAULT = True
+
+    # - Django Pagination
+    PAGINATION_INVALID_PAGE_RAISES_404 = True
+    PAGINATION_DEFAULT_WINDOW = 2
+    PAGINATION_DEFAULT_MARGIN = 1
+
+    # Logging
+    LOGGING = {
+        "version": 1,
+        "disable_existing_loggers": True,
+        "formatters": {
+            "verbose": {
+                "format": "%(levelname)s %(asctime)s %(module)s "
+                "%(process)d %(thread)d %(message)s"
+            }
+        },
+        "handlers": {
+            "console": {
+                "level": "DEBUG",
+                "class": "logging.StreamHandler",
+                "formatter": "verbose",
+            }
+        },
+        "loggers": {
+            "django.db.backends": {
+                "level": "ERROR",
+                "handlers": ["console"],
+                "propagate": False,
+            }
+        },
+    }
+
+    # Demo
+    RICHIE_DEMO_SITE_DOMAIN = values.Value(
+        default="localhost:8080", environ_name="RICHIE_DEMO_SITE_DOMAIN", environ_prefix=None
```

Comment:
When PR https://github.com/openfun/richie-site-factory/pull/274 is merged and you rebase, it should be port 8000 + other modifications as done in pointed PR for other sites.

Reply:
So i wait for this PR merge and i will just have to change 8080 to 8000 ?

Reply:
It is now merged. You need to change the port and also do all the modifications that this PR is doing to all projects.